universal-application-tool-0.0.1/app/views/questiontypes/RepeaterQuestionRenderer.java

```diff
@@ -20,8 +35,11 @@ public RepeaterQuestionRenderer(ApplicantQuestion question) {
 
   @Override
   public Tag render() {
+    ContainerTag repeaterFields = div().withId(REPEATER_FIELDS_ID);
+    // TODO: add each answer as a repeaterField
+    repeaterFields.with(repeaterField(PLACEHOLDER, Optional.empty(), 0));
```

Comment:
question (blocking): if an applicant initially had no entities (ex: they have no children) but later need to add one, how will this work if there is no initial field? Or will we always display a blank one?

Reply:
I wasn't sure what we wanted the UX to be.

I think showing a blank one makes sense. I think this would also make sense for the `MultiOptionQuestionDefinition` piece too since the admin wouldn't ever create a question without giving it at least one option.

Not sure how `value=null` shows up in the request object, but we'd be able to handle things on the server side.

Reply:
Thanks for the explanation!